lib/rename-in-element.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = class RenameInElement {
+  transform(ast) {
+    this.syntax.traverse(ast, {
+      BlockStatement(node) {
+        if (node.path.original === 'in-element') {
+          node.path.original = '-in-element';
+          node.path.parts = ['-in-element'];
+
+          let insertBeforeHash = node.hash.pairs.find(pair => pair.key === "insertBefore");
+          // Somehow `insertBefore=undefined` gets effectively passed when invoking `{{#in-element dest}}`
+          // which later throws in Ember's own AST transform, so removing this here.
+          if (insertBeforeHash && insertBeforeHash.value.type === 'UndefinedLiteral') {
```

Comment:
👆this is a weird one I don't really understand where this comes from!? 🤔

But not filtering out that somehow passed `insertBefore=undefined` causes Ember's AST transform to throw, see https://travis-ci.org/kaliber5/ember-in-element-polyfill/jobs/641986205#L572.